downloads/swan-lake-release-notes/swan-lake-2201.3.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,468 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.3.0 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201-3-0/
+active: 2201-3-0
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201-3-0
+    - /downloads/swan-lake-release-notes/2201.3.0/
+    - /downloads/swan-lake-release-notes/2201-3-0-swan-lake/
+    - /downloads/swan-lake-release-notes/2201-3-0-swan-lake
+---
+
+## Overview of Ballerina 2201.3.0 (Swan Lake)
+
+<em>2201.3.0 (Swan Lake) is the third major release of 2022, and it includes a new set of features and significant improvements to the compiler, runtime, standard library, and developer tooling. It is based on the 2022R4 version of the Language Specification.</em> 
+
+## Update Ballerina
+
+**If you are already using Ballerina 2201.0.0 (Swan Lake)**, run either of the commands below to directly update to 2201.3.0 using the [Ballerina Update Tool](/learn/cli-documentation/update-tool/).
+
+`bal dist update` (or `bal dist pull 2201.3.0`)
+
+**If you are using a version below 2201.0.0 (Swan Lake)**, run the commands below to update to 2201.3.0.
+
+1. Run `bal update` to get the latest version of the Update Tool.
+
+2. Run `bal dist update` ( or `bal dist pull 2201.3.0`) to update your Ballerina version to 2201.3.0.
+
+However, if you are using a version below 2201.0.0 (Swan Lake) and if you already ran `bal dist update` (or `bal dist pull 2201.3.0`) before `bal update`, see [Troubleshooting](/downloads/swan-lake-release-notes/swan-lake-2201.0.0#troubleshooting) to recover your installation.
+
+## Install Ballerina
+
+If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
+
+## Language updates
```

Comment:
@hasithaa / @gimantha Why we don't have a `Bug fixes` section for `Language updates`?  Seems we have closed 50 bugs? https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.3.0+label%3AType%2FBug+is%3Aclosed

Reply:
Added.